Sources/ApiVideoLiveStream.swift

```diff
@@ -64,6 +69,19 @@ public class ApiVideoLiveStream {
         }
     }
 
+    // TODO: Fix zoom flow so it can be a React prop instead of method
+    public var zoomEnabled: Bool = false {
```

Comment:
maybe rename to `pinchedZoomEnabled` instead.
We might want to directly have a `setZoomFactor` API. `zoomEnabled` won't have sense in this case.

Reply:
Done